app/controllers/admins/semesters_controller.rb

```diff
@@ -40,6 +40,7 @@ def destroy
       @semester = Semester.find params[:id]
       if @semester.can_be_destroyed?
         @semester.destroy
+        flash[:error] = "Semester has been successfully deleted"
```

Comment:
Since the semester was successfully deleted, why don't we use `flash[:success]`?

Also, since we're using I18n for the error message below, might as well use it here as well, so something like `t("admin.semester.destroy.success")`